pkg/api/azenvtypes.go

```diff
@@ -138,18 +138,18 @@ var (
 	// AKSWindowsServer2019OSImageConfig is the aks-engine image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-smalldisk-2203",
+		ImageSku:       "2019-datacenter-core-smalldisk-2204",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.2565.220301",
+		ImageVersion:   "17763.2565.220408",
 	}
 
 	// AKSWindowsServer2019ContainerDOSImageConfig is the aks-engine image based on Windows Server 2019
 	// configured with containerd
 	AKSWindowsServer2019ContainerDOSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-ctrd-2203",
+		ImageSku:       "2019-datacenter-core-ctrd-2204",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.2565.220301",
+		ImageVersion:   "17763.2565.220407",
```

Comment:
I assume it's intentional that these two images have a different image version? (built on subsequent days?)

Reply:
Kind of... the CI job which added the images to the marketplace office ran on different days for the two images and that timestamp is generated by the script that does the publishing.

Reply:
so not intentional but by design?

Reply:
All good, thanks for clarifying. Programmatic naming is good. :)